descqa/shear_test.py

```diff
@@ -342,9 +359,9 @@ def run_on_single_catalog(self, catalog_instance, catalog_name, output_dir):
                     sig_jack[i] = np.sqrt(gg.varxi[i])*1.e6
                     sigm_jack[i] = np.sqrt(gg.varxi[i])*1.e6
 
-            n_z = catalog_data[self.z][zmask]
+            n_z = catalog_data[self.z][zmask*mask_mag]
```

Comment:
use ` & ` instead of `*` to join boolean masks. or use pre-defined mask as discussed above.